### DISCUSSION TITLE
Unable to visualized waveform when recording in react.

### DISCUSSION THREAD

**Discussion by ParamPragyan** [katspaugh/wavesurfer.js#3772] [Q&A]
I want to visualize the audio while recording but I am only able to load the audio after recording.
Please help me out.

This is my code
![carbon (6)](https://github.com/katspaugh/wavesurfer.js/assets/54448525/190009de-c51a-4cdf-9131-5782c7e48934)

**Comment by entonbiba** [comment]
@ParamPragyan please provide the code as text not an image to test,
